cloudant-client/build.gradle

```diff
@@ -0,0 +1,129 @@
+//note the gradle project name (cloudant-client) is the maven artifactId
+//the maven pom name entry and User-Agent name in client.properties are defined by clientName
+ext.clientName = 'java-cloudant'
+
+// Additional configurations for javadoc
+configurations {
+    linkableJavadoc {
+        transitive false
+    }
+}
+
+dependencies {
+    compile project(':cloudant-http')
+    linkableJavadoc project(':cloudant-http')
+    compile group: 'commons-codec', name: 'commons-codec', version: '1.6'
```

Comment:
I have declared the dependencies only once in the appropriate sub-projects in e6b34cf250b7c471dca7c7df214c66d506477f55.
As discussed I have also removed the GSON dependency from the http artifact in 0fbd32dc2759b4e2eda7a30ab1d19804892ef97e.